prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -3418,6 +3418,99 @@
               valueType: String
               hl7Use: XTN.12
 
+- Resource: Any
```

Comment:
As noted in standup, this pattern is somewhat cobbled together from other extension mapping patterns because of its uniqueness. 

CP is somewhat unique in that:
- It does not exist in the mapping inventory
- It is a datatype that exists on multiple HL7 segments

With that in mind, the idea here is to create something that is more formalized than simply adding each field to Resource.extension by itself. This allows CP to be noted as such when it shows up, and makes it easier to send CP data if a FHIR sender ever needed to do so.

Still, this is what made sense to me, so if there is a format that better fits existing/future patterns I'm open to it.